.github/workflows/tests.yml

```
@@ -7,21 +7,23 @@ on:
   pull_request:
 
 jobs:
-  test:
+  run-tests:
+    name: lint-and-test
```

small nit: it sounds better if we use more generic way - since we're doing test and lint and later we might do even more (like format or SCA) 
```suggestion
  tests:
```